source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
```

Comment:
@davidhesselbom of course, you're right, memcpy-ing one byte isn't the most efficient operation :) should be possible to set a size threshold somehow, but maybe I'm micro-optimizing now.